bindata/cloud-network-config-controller/managed/controller.yaml

```diff
@@ -165,10 +165,6 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        - name: KUBERNETES_SERVICE_PORT
```

Comment:
Note: `KUBERNETES_SERVICE_PORT` and `KUBERNETES_SERVICE_HOST` were not used in this container anyway.
CNCC uses a kubeconfig that is build here: https://github.com/openshift/cluster-network-operator/pull/2182/files#diff-2a7bd982b6883fe7152cf23cef357f7d9e2ee3c59d1cce19d1666d7939ae5c37L72-L90
The env vars used point to the local API service: https://github.com/openshift/cluster-network-operator/blob/1bb16ff35e90ad2e610fed12ea29e459c879c850/pkg/bootstrap/types.go#L123-L126